tests/data/data_dir_with_no_manifests/configuration/values.json

```diff
@@ -0,0 +1,3 @@
+{
+    "n_iterations": 5
+}
```

Comment:
@cortadocodes have you done `pre-commit install`? If so, then our end-of-file fixer isn't working for json files.

Reply:
I think I may not have done for `octue-sdk-python` - I'll do it now

Reply:
It seems to be skipping all the checks when I run `pre-commit`?